roles/prometheus/templates/prometheus.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus
+Wants=network-online.target
+After=network-online.target
+[Service]
+User=prometheus
+Group=prometheus
+Type=simple
+ExecStart=/etc/prometheus/prometheus \
```

Comment:
Should the paths in ExecStart use the role variables (`prometheus_directory` and `prometheus_tsdb_directory`)?

Reply:
Added